sentry_sdk/integrations/opentelemetry/span_processor.py

```diff
@@ -53,13 +52,11 @@ def link_trace_context_to_error_event(event, otel_span_map):
         return event
 
     ctx = otel_span.get_span_context()
-    trace_id = format_trace_id(ctx.trace_id)
-    span_id = format_span_id(ctx.span_id)
 
-    if trace_id == INVALID_TRACE_ID or span_id == INVALID_SPAN_ID:
```

Comment:
`INVALID_TRACE_ID` and `INVALID_SPAN_ID` are [integers](https://github.com/open-telemetry/opentelemetry-python/blob/2073c36b411b5bddf04f470ed8ece15de59f1920/opentelemetry-api/src/opentelemetry/trace/span.py#L578-L579) while `trace_id` and `span_id` are strings, so this would never be `True`.